.changeset/swift-planets-try.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+This Pr is separating sources in testmo for playwright and cypress tests, and adding results for runs on PR to testmo
```

Comment:
Use more explicit language what is the result of your change and how that impacts others:

It could be something like:

_Testmo results are no longer have fixed source name. This means the report names in testmo has separated run names depending on its source (cypress/playwright)_